scripts/vite.config.js

```diff
@@ -1,16 +1,39 @@
-import { defineConfig } from 'vite'
-import { svelte } from '@sveltejs/vite-plugin-svelte'
-import dts from 'vite-plugin-dts'
+import { defineConfig } from 'vite';
+import { svelte } from '@sveltejs/vite-plugin-svelte';
+import dts from 'vite-plugin-dts';
+import { Eta } from 'eta';
 
 const buildLib = process.env.BUILD_LIB;
 
+/**
+ * @returns {import('vite').Plugin}
+ */
+function eta() {
+  const eta = new Eta({
+    autoTrim: false
+  });
+
+  return {
+    transform(template, id) {
+      if (id.match(/.eta$/)) return {
```

Comment:
```suggestion
      if (id.match(/\.eta$/)) return {
```
Forgot to change this, unescaped `.` matches any character.